pkgs/top-level/all-packages.nix

```diff
@@ -20334,6 +20334,7 @@ with pkgs;
   prometheus-dnsmasq-exporter = callPackage ../servers/monitoring/prometheus/dnsmasq-exporter.nix { };
   prometheus-dovecot-exporter = callPackage ../servers/monitoring/prometheus/dovecot-exporter.nix { };
   prometheus-domain-exporter = callPackage ../servers/monitoring/prometheus/domain-exporter.nix { };
+  prometheus-fastly-exporter = callPackage ../servers/monitoring/prometheus/fastly-exporter.nix { };
```

Comment:
Please also add a test.

Reply:
Thanks for the review!
I don't think a test is feasible for this since it requires networking access to api.fastly.com and a valid token.

I'll add the module :+1: .